roles/openshift_sanitize_inventory/tasks/__deprecations_logging.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I don't like sticking this logic in openshift_sanitize_inventory.  I think this logic should either be placed inside the role or the play in which we call the role.  Placing this logic here is confusing, and it's not specifically tied to the logging role so may go stale.

Also, sticking in some debug messages during the run., nobody's going to see that among the other 40k lines of stuff that scrolls by, even with a 5 second pause.  I open a terminal to kick off installs and then go do something else while it runs.

The advantage to sticking the deprecation parts inside the role are that it lives with the role, and users wanting to make changes will be consulting that role's logic anyway.

Reply:
What about something like this? https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/validate_hostnames.yml#L11-L20 it's modal unless you override it

Reply:
@sdodson would we want to have that flag just skip the pause or the warning message too?

Reply:
Just the pause that way the logs would still have an accurate list of the deprecated variables if they were to look for them.

Reply:
Maybe we can also print a summary at the end too?

Reply:
Now that I can get it to run to completion without error I'll try to figure this out

Reply:
@sdodson I'm not sure how to get this to display at the end. any thoughts what approach I can take? A handler doesn't seem to be firing..

Reply:
A handler would only fire at the end of a play anyway. That's fine as long as your play is the last one... otherwise I think you want a callback plugin.